CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ## 4.0.0-alpha.2
 
 - feat: add contexts to scope
+- feat: add missing protocol classes : Request, Mechanism
```

Comment:
I'd mention the other classes too or remove the specification (Request, Mechanism)

Reply:
✅